Source/Renderer/Context.js

```diff
@@ -312,6 +331,15 @@ define([
             glVertexAttribDivisor = function(index, divisor) { gl.vertexAttribDivisor(index, divisor); };
 
             glDrawBuffers = function(buffers) { gl.drawBuffers(buffers); };
+
+            createQuery = function() { return gl.createQuery(); };
```

Comment:
Can you update [getWebGLStub.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/getWebGLStub.js) with the new timer functions?